forms-html/Shirin-MohammedKhalil/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Shirin-Mohammed</title>
+  </head>
+  <body>
+    <form action="#" method="POST">
+      <!-- Email-->
+      <label for="email">Enter Your E-mail </label><br />
+      <input
+        type="email"
+        id="email"
+        name="email"
+        required
+        placeholder="Shirin-Mohammed@gmail.com"
+      />
+      <br />
+
+      <!-- User Name-->
+      <label for="username"> User Name</label><br />
+      <input
+        type="text"
+        id="username"
+        name="username"
+        placeholder="Shirin - Mohammed"
+      />
+      <br />
+
+      <!-- Password-->
+      <label for="password">Password</label> <br />
+      <input
+        type="password"
+        id="password"
+        name="password"
+        placeholder="***********"
+      />
+      <br />
+
+      <!-- Quantity-->
+      <label for="quantity"> Quantity (between 1 - 5): </label>
+      <input type="number" id="quantity" name="quantity" min="1" max="5" />
+      <br />
+
+      <!-- Volume-->
+      <p>Volume (between 0 and 50):</p>
+      <input type="range" name="volume" min="0" max="50" /> <br />
+
+      <!-- Radio-->
+      <label for="radio"> Radio</label>
+      <br />
+      <input type="radio" id="male" name="gender" value="male" />
+      <label for="male">Male</label><br />
+      <input type="radio" id="female" name="gender" value="female" />
+      <label for="female">Female</label><br />
+      <input type="radio" id="other" name="gender" value="male" />
+      <label for="other">Other</label><br /><br />
+
+      <!-- Checkbox-->
+      <p>Checkbox</p>
+      <br />
+      <input type="checkbox" id="bike" name="bike" value="bike" />
+      <label for="bike">I have a bike</label><br />
+      <input type="checkbox" id="car" name="car" value="car" />
+      <label for="car">I have a Car</label><br />
+      <input type="checkbox" id="boat" name="boat" value="boat" />
+      <label for="boat">I have a boat</label><br /><br />
+      <br />
+
+      <!-- Button-->
+      <p>button</p>
+      <br />
+      <input type="button" name="button" value="Click Me!" />
+      <br />
+
+      <!-- Color-->
+      <p>Select Your Favorite Color</p>
+      <input type="color" name="color" /><br />
+
+      <!-- 1980 > Date-->
+      <p>Enter a date before 1980-01-01:</p>
+      <br />
+      <input type="date" name="date" max="1980-01-01" /><br />
```

Comment:
Props on enforcing the user to enter a date before 1980-1-1 by using the max attribute.